OpenRPA.NativeMessagingHost/myAddon2/content.js

```diff
@@ -386,12 +413,51 @@ if (true == false) {
                         }
                     }, true);
                     document.addEventListener('change', function (e) {
-                        if (inputTypes.some(i => i === e.target.type)) {
-                            openrpautil.pushEvent('change', e);
-                        }
+                        handleChange(e);
                     }, true);
                     document.addEventListener('mousedown', function (e) { openrpautil.pushEvent('mousedown', e); }, true);
 
+                    const mutationObserver = new MutationObserver((mutations) => {
```

Comment:
A small, little, detail: since we already have another `MutationObserver` can you rename this to shadowRootsObserver and the already present into iframeObserver? Thanks!